site/docs/components/border-layout/usage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### When to use Border Layout
+
+- To manage the top-level layout of your application, or page, with at least one additional region alongside the main content area (e.g., for navigation or a footer).
+
+### When not to use Border Layout
+
+- You’re not managing the top-level layout of your application, but the layout inside one of the five Border Layout regions.
+
+- For a customized layout that you can control in both row and column dimensions (for example, a Dashboard). Instead, use Grid Layout.
+
+- For a side panel which temporarily reveals relevant content, consider using a Drawer component instead of a border layout region.
```

Comment:
```suggestion
- For a side panel that temporarily reveals relevant content, consider using a Drawer component instead of a border layout region.
```